org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corext/template/java/JavaPostfixContext.java

```diff
@@ -316,16 +316,14 @@ public boolean canEvaluate(Template template) {
 		if (isForceEvaluation())
 			return true;
 
-		if (selectedNode == null) // We can evaluate to true only if we have a valid inner expression
-			return false;
-
 		if (template.getName().toLowerCase().startsWith(getPrefixKey().toLowerCase()) == false) {
 			return false;
 		}
 
-		if (selectedNode instanceof Javadoc) {
+		// We can evaluate to true only if we have a valid inner expression
+		// Do not evalute within Javadoc elements
+		if (selectedNode == null || selectedNode instanceof Javadoc)
```

Comment:
Move to L319, it's the faster check

Reply:
Right. `getPrefixKey` involves the document itself so better to check before that.